packages/sei-integration-tests/src/module.rs

```diff
@@ -187,6 +187,14 @@ impl Module for SeiModule {
                     id,
                 );
             }
+            // TODO: Implement get denom authority metadata in integration tests
```

Comment:
As mentioned, Integration test additions for tokenfactory will occur in separate PR which will refactor integration tests in sei-cosmwasm